### DISCUSSION TITLE
Google virus detection?

### DISCUSSION THREAD

**Discussion by sevenreeds** [nextcord/nextcord#370] [Q&A] [answered]
Hi,

I have an odd problem.  In the last few weeks I have switched from `discord.py` to `nextcord`.  The bot I have is trivial. A friend asked me to create it for his server.  I use `pinstaller` to create a Win-10 executable.  I bundle the `dist` result into a zip archive.  This is stored on a shared Google Drive.  Once on the shared drive Google flags it as containing a virus.

I have checked my primary linux desktop and the Win-10 VM, in which I build the executable.  No viruses detected.

This is a new problem for me.  I am not sure how to investigate this.  I am looking for pointers and advice.  I am not claiming that `nextcord` is the issue, it is just the newest item in the mix.

Cheers

**Comment by EpicBirb** [comment, marked as answer]
If I remember correctly, PyInstaller will be flagged as a virus for some weird reason. I have tried this with my own project and it got flagged. I read somewhere online that you can switch to a development branch of Pyinstaller and the file will not be flagged. ¯\_(ツ)_/¯
